tests/cli-configuration.spec.js

```diff
@@ -265,4 +266,14 @@ describe('CLI Configuration', function() {
 
   });
 
+  describe('.load()', function() {
+
+    it(`should load defualt internal ${packageJSON.name}.conf.js if file is not found`, function() {
```

Comment:
I think that's a fair point. If we're specifying a config file and it doesn't exist, we should error. If no config file is specified, and there's no `versionist.conf.js` in the CWD, then we should fall back to `versionist`s internal config.